src/common/string_util.h

```diff
@@ -92,6 +128,15 @@ bool parse_int(const std::string_view str, T& value, int base = 10) {
   return true;
 }
 
+/// Converts an integer value into a string via std::to_chars (i.e. without locale).
+template <typename T, typename = std::enable_if_t<std::is_integral_v<T>>>
+std::string int_to_string(const T& value, int base = 10) {
+    char buf[8*sizeof(T) + std::is_signed_v<T>]; // maximum possible size with smallest possible base (2)
+    auto [p, ec] = std::to_chars(std::begin(buf), std::end(buf), value, base);
```

Comment:
if std::to_chars returns an error in release mode this could return buf as uninitialized memory into a string, can we value initialized buf?